core/src/main/java/io/questdb/ClientTlsConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package io.questdb;
```

Comment:
This file seems to be missing the license header. Same story with other files introduced by the PR.

Reply:
thanks!